backend/src/services/linear/api/resolvers.py

```
@@ -108,6 +159,312 @@ def resolve_issue_labels(
+    session: Session = info.context["session"]
+
+    # Build base query for comments on this issue
+    base_query = session.query(Comment).filter(Comment.issueId == issue.id)
+
+    # Filter archived comments unless includeArchived is True
+    if not includeArchived:
+        base_query = base_query.filter(Comment.archivedAt.is_(None))
+
+    # Apply custom filter if provided
+    if filter:
+        base_query = apply_comment_filter(base_query, filter)
```

Ruff (0.14.4)

Undefined name `apply_comment_filter`

(F821)